core/storage/question/gae_models.py

```diff
@@ -135,9 +129,8 @@ def create(
         question_model_instance = cls(
             id=instance_id,
             question_state_data=question_state_data,
-            question_state_data_schema_version=(
-                question_state_data_schema_version),
-            language_code=language_code)
+            language_code=language_code,
+            version=feconf.CURRENT_QUESTION_VERSION)
```

Comment:
This is not needed, see my other comment. I think you can pass in version?

Reply:
Okay done